Sources/Build/BuildOperationBuildSystemDelegateHandler.swift

```diff
@@ -757,10 +757,11 @@ final class BuildOperationBuildSystemDelegateHandler: LLBuildBuildSystemDelegate
     }
 
     /// Invoked when an action taken before building emits output.
-    func preparationStepHadOutput(_ name: String, output: String) {
+    /// when verboseOnly is set to true, the output will only be printed in verbose logging mode
```

Comment:
formatting nit, which will make it render nicely in editor previews and tools like DocC
```suggestion
    /// - Parameters:
    ///   - name: name of the preparation step
    ///   - output: output from the preparation step delegated to this handler
    ///   - verboseOnly: when set to `true`, the output will only be printed in verbose logging mode
```

Reply:
this is an internal function, so not super important?